maths/find_min.py

```diff
@@ -1,3 +1,12 @@
+"""
+Referencing the Big O Notation, this algorithm has a liner time complexity, O(N).
```

Comment:
```suggestion
Referencing the Big O Notation, this algorithm has a liner time complexity, O(n).
```
I believe the big O notation uses lowercase `n`